Week1/homework/js-exercises/TypeChecker.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+//declare variable
+let firstName = 'ahmed';
+let lastName = 'shahwan';
+
+// declare 2 objects variable
+let address = {
+  postCode: '1214hr',
+  city: 'Hilversum',
+};
+let dateOfBirth = {
+  year: 1988,
+  month: 'march',
+  day: 10,
+};
+
+//print the data type of the variable
+console.log(typeof firstName);
+console.log(typeof lastName);
+console.log(typeof address);
+console.log(typeof dateOfBirth);
+
+//set 6 conditional statements using if else
+if (typeof firstName === typeof lastName) {
+  console.log('Same  Type');
+} else {
+  console.log('NOT the Same');
+}
```

Comment:
Using shorthand (inline) could help you write even shorted statements.

```
check ? true : false
```

So in this case for instance:

```
console.log(typeof firstName === typeof lastName ? 'Same type' : 'Differtent type');
```